Objects/Objects/Geometry/Pointcloud.cs

```diff
@@ -27,9 +27,18 @@ public class Pointcloud : Base, IHasBoundingBox, ITransformable<Pointcloud>
     public string units { get; set; }
 
     public Pointcloud()
+    
+    /// <param name="points">Flat list of x,y,z coordinates</param>
+    /// <param name="colors">Optional list of colors</param>
+    /// <param name="sizes">Optional list of sizes</param>
+    [SchemaInfo(nameof(Pointcloud), "Create a point cloud object", "Objects", "Geometry")]
+    public Pointcloud(List<double> points, List<int> colors = null, List<double> sizes = null)
```

Comment:
Seems like `colors` is not being used?

Reply:
ah good catch, didn't see those lines missing. just pushed the fix